packages/core/src/repositories/runErrors/evaluationResultsRepository/index.test.ts

```diff
@@ -105,6 +106,7 @@ describe('EvaluationResultsRepository', () => {
       await createRunError({
         errorableType: ErrorableEntity.EvaluationResult,
         errorableId: evaluationResult2.id,
+        errorableUuid: evaluationResult2.uuid,
```

Comment:
:\ you are editing the tests but not the code?

Reply:
This interactor is not used, only here in these tests. I made NOT NULL `errorableUuid` so this was complaining